src/org/thoughtcrime/securesms/loki/redesign/messaging/LokiPublicChatPoller.kt

```diff
@@ -235,22 +236,24 @@ class LokiPublicChatPoller(private val context: Context, private val group: Loki
                 }
             }
             Promise.of(messages)
-        }.successBackground {
+        }.success {
             val newDisplayNameUpdatees = uniqueDevices.mapNotNull {
                 // This will return null if the current device is a master device
                 LokiDeviceLinkUtilities.getMasterHexEncodedPublicKey(it).get()
             }.toSet()
             // Fetch the display names of the master devices
             displayNameUpdatees = displayNameUpdatees.union(newDisplayNameUpdatees)
-        }.successBackground { messages ->
+        }.success { messages ->
```

Comment:
`successBackground` does the same thing as what you just did below

Reply:
I know. Was trying to make it more explicit what's going on.